common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -4,7 +4,8 @@
 @import templates.inlineJS.blocking.js._
 
 <!--[if lt IE 9]>
-    <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
+    <script src="@Static("javascripts/ie8polyfills.js")"></script>
+    <script>@Html(polyfills().body)</script>
```

Comment:
Doing it right now. Also I think JSON is also an IE7- issue so I'll move it into ie8polyfills.